### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Minecodes**
I added a version to run the script with one command

### CODE REVIEW

**Comment by Minecodes** [comment]
Normal shell doesn't work, but bash

File: README.md [hunk 1]
```diff
@@ -6,3 +6,8 @@ Simple Minecraft Server Install Script (Linux)
 - Give yourself permission with `chmod +x minecraftsi.sh`
 - Run it with `./minecraftsi.sh`
 - You might have to install the `dialog` package to use.
+
+or
```
Comment (MrSlimeDiamond) [inline README.md:9]: This could be changed to something along the lines of "Or alternatively (only on bash)"

File: README.md [hunk 2]
```diff
@@ -6,3 +6,8 @@ Simple Minecraft Server Install Script (Linux)
 - Give yourself permission with `chmod +x minecraftsi.sh`
 - Run it with `./minecraftsi.sh`
 - You might have to install the `dialog` package to use.
```
Comment (MrSlimeDiamond) [inline README.md]: Maybe remove this line and put it at the bottom

File: README.md [hunk 3]
```diff
@@ -6,3 +6,8 @@ Simple Minecraft Server Install Script (Linux)
...
+
+or
+
+- Run script with `sh -c "$(curl -fsSL https://raw.githubusercontent.com/MrSlimeDiamond/minecraftsi/main/minecraftsi.sh)"` or `bash -c "$(curl -fsSL https://raw.githubusercontent.com/MrSlimeDiamond/minecraftsi/main/minecraftsi.sh)"`
+- You might have to install the `dialog` package to use.
```
Comment (MrSlimeDiamond) [inline README.md]: Remove the `-` from this and put it a line down please

**Review by MrSlimeDiamond — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -5,4 +5,11 @@ Simple Minecraft Server Install Script (Linux)
...
 - Run it with `./minecraftsi.sh`
-- You might have to install the `dialog` package to use.
+
+or
+
+Run script with `bash -c "$(curl -fsSL https://raw.githubusercontent.com/MrSlimeDiamond/minecraftsi/main/minecraftsi.sh)"`
+
+## Requirements
+
+- `dialog` - installable from apt and yum
```
Comment (MrSlimeDiamond) [inline README.md]: "Installable via your package manager"

**Review by MrSlimeDiamond — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Minecodes) [reply README.md]: How do you mean it?

Comment (MrSlimeDiamond) [reply README.md]: To change it to something along those lines

**Pull request merged**